elevate-cpanel

```diff
@@ -7572,6 +7595,50 @@ EOS
         return;
     }
 
+    sub get_extra_packages ($self) {
+        my $pkgmgr = $self->pkgmgr;
+
+        my $out   = $self->cpev->ssystem_hide_and_capture_output( $pkgmgr, 'list', 'extras' );
+        my @lines = @{ $out->{stdout} };
+        while ( my $line = shift @lines ) {
+            last if $line && $line =~ m/^Extra Packages/;
+        }
+
+        my @extra_packages;
+        while ( my $line = shift @lines ) {
+            chomp $line;
+            my ( $package, $version, $repo ) = split( qr{\s+}, $line );
+
+            if ( !length $version ) {
+                my $extra_line = shift @lines;
+                chomp $extra_line;
+                $extra_line =~ s/^\s+//;
```

Comment:
Interesting that we'd even get word wrap here. Only makes sense in the context that we unshift `/usr/bin/bash -c` to our arglist in `_ssystem`, which arguably should be optional/overridable to avoid the need for shenanigans like this.

Or... since we are running via bash, you could always run `stty cols 256 && cmd` or something similarly large to avoid wrap ever happening maybe?